extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -21,15 +21,21 @@ amp-story-shopping-tag {
   line-height: 15.4px !important;
 }
 
-/* Maintains the dot position in RTL mode */
+/* Maintains the correct image and text order inside the tag in RTL mode */
 [dir=rtl].i-amphtml-amp-story-shopping-tag-inner {
   flex-flow: row-reverse !important;
 }
 
+/* Maintains the dot position in RTL mode */
 [dir=rtl] .i-amphtml-amp-story-shopping-tag-dot {
   margin-inline-start: 4px !important;
 }
 
+/* Maintans the correct margins in RTL mode */
+[dir=rtl] .i-amphtml-amp-story-shopping-tag-pill {
+  padding: 4px 4px 4px 12px !important
+}
```

Comment:
We can use inline margins and remove these two rtl blocks.
Check out example code here:
https://github.com/ampproject/amphtml/pull/37576/files

As a rule of thumb, don't mix margins and padding. 
Currently `i-amphtml-amp-story-shopping-tag-pill` has padding and the child `i-amphtml-amp-story-shopping-tag-pill-image` has margin.
It makes maintenance difficult and it can be solved more simply by ether putting the value on the parent or the child. It would be okay to do that clean-up in this PR since it's related to the elements being worked on.

Reply:
Sounds good, I'll keep it in mind for future CSS PRs.